addsvc/httpjson_binding.go

```diff
@@ -0,0 +1,80 @@
+package main
+
+import (
+	"encoding/json"
+	"io"
+	"net/http"
+	"time"
+
+	"golang.org/x/net/context"
+
+	"github.com/peterbourgon/gokit/metrics"
+	"github.com/peterbourgon/gokit/server"
+	"github.com/peterbourgon/gokit/server/zipkin"
```

Comment:
strong suggestion to make the tracing support pluggable... happy to elaborate/justify as desired!